bin/gwd/gwd.ml

```diff
@@ -1109,6 +1121,11 @@ let make_conf from_addr request script_name env =
       if x = "" then !default_lang else x
     with Not_found -> !default_lang
   in
+  let vowels =
+    try List.assoc "vowels" base_env with
```

Comment:
you can use `List.assoc_opt` to avoid try ... catch

Reply:
ok